### ISSUE TITLE
Interior zones reporting nighttime solar gains in simulation with interior windows

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#7865]
Issue overview
--------------
With interior windows, there are significant non-zero values of Surface Inside Face Solar Radiation Heat Gain Energy at night when they should all be zero for interior zones.. (This is not related to the airwall work, similar numbers show in v9.1, probably need a new issue for this).

Happens with both FullExterior and FullInteriorExterior solar distribution.

Test file is based on 5ZoneAirCooled. There are interior windows between zones 1 and 4, 1 and 5, 4 and 5. Surfaces in zones 4 and 5, for example are showing solar gains 24/7.
e.g. column GV "F5-1:Surface Inside Face Solar Radiation Heat Gain Energy [J](Hourly)" (floor of zone 5, the core zone).

### Details
Some additional details for this issue (if relevant):
 - Platform Win10
 - Version of EnergyPlus 9.3.0-https://github.com/NREL/EnergyPlus/commit/fae01bd1265f5eed24abcdd92cdf3e2626e5089f

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file attached (run with Chicago epw): 
[7865-FullExt-5ZoneAirCooled_Daylighting-ActualIntWindow.idf.txt](https://github.com/NREL/EnergyPlus/files/4345899/7865-FullExt-5ZoneAirCooled_Daylighting-ActualIntWindow.idf.txt)
[7865-FullIntExt-5ZoneAirCooled_Daylighting-ActualIntWindow.idf.txt](https://github.com/NREL/EnergyPlus/files/4345900/7865-FullIntExt-5ZoneAirCooled_Daylighting-ActualIntWindow.idf.txt)

 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by RKStrand** [comment]
@mjwitte Interesting.  I updated the files and ran one.  I can confirm that there is a non-zero value on the floor as you pointed out.  Before I got any results though, I got a hard crash in EnergyPlus.  The call to DayltgGlareWithIntWins was sending the wrong variable.  I'll just drop that fix in along with whatever I discover here.

**Comment by RKStrand** [comment]
@mjwitte I think that I have figured out why there are zero values when the sun is down.  When I zeroed out all the lights, this problem went away.  Looking through the code, the SW that it reports is the solar minus the lights from the zone in question.  However, there is also visible coming from lights in other zones.  I have tested this out by calculating the light from other zones and subtracting it out of the solar like the lights for the current zone are subtracted out.  Now, the solar at night is showing up as zero like it should.  There are lights on at night--that is also reported.  Adding some new variables to report the light from other zones getting through an interzone window that is incident on surfaces.  It seems like this is headed in the right direction.
